packages/assets-controllers/src/TokenDetectionController.test.ts

```diff
@@ -164,22 +161,24 @@ describe('TokenDetectionController', () => {
 
     preferences = new PreferencesController({}, { useTokenDetection: true });
     controllerMessenger = getControllerMessenger();
-    const networkSetup = setupNetworkController(controllerMessenger);
-    network = networkSetup.network;
-    networkMessenger = networkSetup.networkMessenger;
+    sinon
```

Comment:
Nit: Typically it's better to leave stubs as per-test, so that it's more clear why we need it and which test relies on it. Was there a problem with leaving it in-place?

Doesn't really matter here though I guess, as I'd like to see us eventually update these tests to not use a real tokens controller.  When we do that, this stub can be removed.

Reply:
Noted! 

I think this was a rare case where every test in the suite depended on this method being stubbed out. As you mentioned, the better fix is to not create the controller to begin with though. Hopefully we can get back to this soon!